Lib/lib2to3/main.py

```diff
@@ -218,7 +218,7 @@ def main(fixer_pkg, args=None):
 
     # Initialize the refactoring tool
     avail_fixes = set(refactor.get_fixers_from_package(fixer_pkg))
-    unwanted_fixes = set(fixer_pkg + ".fix_" + fix for fix in options.nofix)
+    unwanted_fixes = {fixer_pkg + ".fix_" + fix for fix in options.nofix}
```

Comment:
Skip this.  I prefer the original for clarity and for parallelism with the previous line.